spec/models/profile_spec.rb

```diff
@@ -323,59 +323,95 @@
     it 'activates a profile after password reset' do
       profile = create(
         :profile,
+        :password_reset,
         user: user,
-        active: false,
-        deactivation_reason: :password_reset,
       )
 
+      expect(profile.activated_at).to eq nil # to change
+      expect(profile.active).to eq false # to change
+      expect(profile.deactivation_reason).to eq 'password_reset' # to change
+      expect(profile.fraud_review_pending?).to eq(false)
+      expect(profile.gpo_verification_pending_at.present?).to eq false
+      expect(profile.initiating_service_provider).to eq nil
```

Comment:
Figured out that `initiating_service_provider` is expected to be nil because they're coming from a password reset email. Minor syntax suggestion:
```suggestion
      expect(profile.initiating_service_provider).to be_nil
```

Reply:
I will use `be_nil`. and I will add a comment with your discovery, thank you!